packages/namekit-react/src/components/IconButton.tsx

```diff
@@ -6,15 +6,28 @@ export interface IconButtonProps extends ButtonProps {
   children: React.ReactNode;
 }
 
+const iconButtonSizeClasses = {
+  small: "nk-p-1",
+  medium: "nk-p-2",
+  large: "nk-p-3",
+};
+
 export const IconButton = React.forwardRef<HTMLButtonElement, IconButtonProps>(
-  ({ children, className, ...props }, ref) => {
+  ({ children, className, size = "medium", ...props }, ref) => {
     const combinedClassName = cc([
-      "nk-inline-flex nk-items-center nk-justify-center",
+      "nk-inline-flex nk-items-center nk-justify-center nk-p-2",
+      iconButtonSizeClasses[size],
       className,
     ]);
 
     return (
-      <Button ref={ref} className={combinedClassName} {...props}>
+      <Button
+        ref={ref}
+        className={combinedClassName}
+        size={size}
+        padding={iconButtonSizeClasses[size]}
```

Comment:
Is it really needed to add this `padding` param? Could this same goal be achieved through the values we pass on the `className` field?